docs/quickstart.ipynb

```diff
@@ -4,20 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't think silencing the numpy warnings is a good idea. Users will be confused when they show up running tardis without this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Ok, discarded.